cmd/loogtui/main.go

```diff
@@ -23,6 +17,10 @@ import (
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/util/homedir"
+	"log"
+	"os"
+	"os/signal"
+	"path/filepath"
```

Comment:
please reformat the imports

Reply:
gofmt didn't change anything

Reply:
hm maybe check your GoLand Settings - std imports should be at the very top, while external imports should be in the middle, internal imports at the very end, e.g.

```
import (
  "fmt"

  "sigs.k8s.io/..."

  "github.com/loog-project/loog/internal/..."
)

Reply:
Okay, that seems to have fixed it. The import ordering should be correct now